File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-2.js
const prompt = require('prompt-sync')()

const num1 = prompt('Digite o primeiro numero positivo:')
const num2 = prompt('Digite o segundo numero positivo:')

console.log(`
1. Média ponderada, com pesos 2 e 3, respectivamente
2. Quadrado da soma dos 2 números
3. Cubo do menor número`)

const opcao = Number(prompt('Escolha uma opção:'))

switch (opcao){
    case 1:
        console.log(`Média Ponderada:${(num1*2+num2*3)/5}`)
        break

    case 2:
        console.log(`Quadrado da soma dos 2 números:${num1**2 + num2**2}`)
        break

    case 3:
        num1<num2 ? console.log(`Cubo do menor número:${num1**3}`):console.log(`Cubo do menor número:${num2**3}`)

        
        break
    
    default:
        console.log('Opção inválida')
        break
}

File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_2.py
num_1 = int(input('Digite o primeiro valor:'))
num_2 = int(input('Digite o segundo valor:'))

print('1. Média ponderada, com pesos 2 e 3 respectivamente\n'
      '2. Quadrado da soma dos 2 números\n'
      '3. Cubo do menor número\n')

opcao = int(input('Escolha uma opção:'))

if opcao < 1 or opcao > 3:
    print('Opção Inválida')
else:
    match opcao:
        case 1:
            print(f'Média ponderada: {((num_1*2)+(num_2*3))/5}')
        case 2:
            print(f'Quadrado da soma: {(num_1**2)+(num_2**2)}')
        case 3:
            if num_1 < num_2:
                print(f'Cubo do menor numero: {num_1**3}')
            else:
                print(f'Cubo do menor numero: {num_2**3}')
                